config/share_notify.yml.prod.sample

```diff
@@ -0,0 +1,3 @@
+---
+host: <%= ENV.fetch('APPS_DHH_PROD_SHARE_NOTIFY_HOST') {'https://staging.osf.io'}%>
```

Comment:
Ok, I suppose this is related to my[ question about the share_notify.yml file](https://github.com/ucsdlib/damspas/pull/208#discussion-diff-65209334) I'm curious then, why we're not using the environment convention in the yml file for this, as we are with the other config files?